.luacheckrc

```diff
@@ -5,12 +5,14 @@ globals = {
 }
 
 read_globals = {
+	"core",
 	"table.copy",
 	"table.indexof",
 	"minetest",
 	"ItemStack",
```

Comment:
While at it could maybe straight update luacheck to use builtin luanti standard?

So instead of adding `core` drop all of this core stuff (lines 8-12 atow) and use luanti builtin std. That'll be `std = "luanti+max"` (or `minetest+max`) basically means luanti core and lua versions from 5.1 to latest including luajit.

Example: https://github.com/mt-mods/pipeworks/blob/master/.luacheckrc but instead of `minetest` builtin std can now also use `luanti` std (minetest is now just alias for luanti standard).

Reply:
Sounds great, thanks. I wasn't aware.

doesn't seem to know the luanti namespace yet though:
> Critical error: in config loaded from .luacheckrc: invalid value of option 'std': unknown std 'luanti'

and using "minetest+max" it doesn't know about ``core``, over at https://github.com/mt-mods/digistuff/pull/64

What am I missing? The workflow files are identical.

Reply:
My local platform has no issues with "luanti+max"

Reply:
Sounds like outdated docker image missing luanti std.

`core`, as it became official, was added to luacheck builtin standard with `luanti` while for `minetest`, even if obvious for devs, it wasn't official and not in standard.

Reply:
Seems there was no full release yet so luacheck was only updated partially (for one pre built docker images haven't been updated yet).

Asked for more complete release at lunarmodules/luacheck/issues/121

There is ghcr.io/lunarmodules/luacheck:master image but it isn't used in lunarmodules/luacheck github action, action is instead using specific release which has to be updated.

Reply:
OK, thanks for investigating. So I guess I revert for now and we keep trying later on.